src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+  const menDiedInCentury = century
+    ? men.filter((man) => century === Math.ceil(man.died / 100))
+    : men;
```

Comment:
we can use filter only once
```suggestion
  const men = century 
     ? people.filter((man) =>  person.sex === 'm' && century === Math.ceil(man.died / 100) 
     :  people.filter(person => person.sex === 'm');
 
```